src/goto-instrument/contracts/dynamic-frames/dfcc_instrument.cpp

```diff
@@ -323,10 +327,13 @@ void dfcc_instrumentt::instrument_function_body(
 
   if(!goto_function.body_available())
   {
-    log.warning() << "DFCC instrumentation: '" << function_id
-                  << "' body is not available. Results may be unsound if the "
-                     "actual function has side effects."
-                  << messaget::eom;
+    // generate a default body `assert(false);assume(false);`
```

Comment:
What are we using for the location information of the generated function bodies? Is there anything sensible we can use?

Reply:
`generate_function_bodies_factory` takes care of this: it will use the source location of the declaration of the function.